### ISSUE TITLE
The event handlers are removed when cloning the image by the Lazy Load script

### ISSUE THREAD

**Issue by silkypress** [Automattic/jetpack#12640]
#### Description

Trying to add a "mouseenter" or "click" event on an image that is afterwards processed by the Lazy Load script fails because the image is cloned and then replaced with the clone. Any jQuery event attached to the original image is removed in the process.

#### Steps to reproduce the issue

When adding the following code to the website:
```
jQuery('img.my-img').on('mouseenter', function() {
   console.log('do something on mouseenter over the image');
});
```
it works if the Lazy Load feature is disabled and fails if enabled.

#### Proposed solution

By creating the image clone with "clone(true)" instead of "clone()", the event handlers are copied along (see the [jQuery clone()](https://api.jquery.com/clone/) function), so the "mouseenter" event attached to the original image will also work on the clone.

Here is a proposed patch:

```
--- js_old/lazy-images.js	2019-06-11 10:19:30.813987718 +0200
+++ js/lazy-images.js	2019-06-11 10:21:02.184551328 +0200
@@ -107,7 +107,7 @@
 
 		srcset = theImage.attr( 'data-lazy-srcset' );
 		sizes = theImage.attr( 'data-lazy-sizes' );
-		theClone = theImage.clone();
+		theClone = theImage.clone(true);
 
 		// Remove lazy attributes from the clone.
 		theClone.removeAttr( 'data-lazy-srcset' ),

```

**Comment by jeherve** [comment]
Thanks for the report. Feel free to submit your patch in a Pull Request, we'll be happy to review and test it!

**Comment by jeherve** [comment]
Fixed in #12694
